docs/docs/get-started/setup-lightdash/get-project-lightdash-ready.mdx

```diff
@@ -58,6 +94,10 @@ To install the Lightdash CLI, run the following on your command line.
 npm install -g @lightdash/cli
 ```
 
+:::danger
```

Comment:
Maybe make this a caution or info instead of a danger? Danger makes me feel like I've done something wrong 😳 
```suggestion
:::info
```

Reply:
lol, looks scary